src/Microsoft.AspNetCore.ResponseCaching/CacheEntry/CacheEntryContext.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.ResponseCaching
+{
+    public class KeyValuePair
```

Comment:
Need a reference type here so I can populate the value. Won't work with a struct.